assessment/services/utils.py

```diff
@@ -88,3 +88,26 @@ def get_assessment_event_from_id(assessment_event_id) -> AssessmentEvent:
         return found_events[0]
     else:
         raise EventDoesNotExist(f'Assessment Event with ID {assessment_event_id} does not exist')
+
+
+def get_active_assessment_event_from_id(event_id):
+    found_events = AssessmentEvent.objects.filter(event_id=event_id)
```

Comment:
Hello, if I may ask, is this function used to get active assignments only, or maybe we will expand it in the future?

Reply:
Hello, Alex. Yes, this function will only be used to get active assignments to support the validation for subscription. I am planning to have a separate function dedicated to retrieve events and tools for creating the attempt in the near future.

Reply:
Okay, Jo. Thanks for the clarification